examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p><code>f</code> is still initialised at this point right? No need to fill. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
F has the values from previous case needs to reset to zero or will have imprint of the previous case